gum/backend-linux/gumprocess-linux.c

```diff
@@ -289,6 +291,11 @@ static gssize gum_libc_syscall_4 (gsize n, gsize a, gsize b, gsize c, gsize d);
 
 static gboolean gum_is_regset_supported = TRUE;
 
+#if defined (HAVE_MIPS)
+static int getcontext (ucontext_t *ucp);
+static int setcontext (const ucontext_t *ucp);
```

Comment:
Also, we should move these above the variable, as function prototypes should always come before variables.